### DISCUSSION TITLE
Having to install all peerDeps of tiptap/core

### DISCUSSION THREAD

**Discussion by JClackett** [mantinedev/mantine#3176] [Q&A]
### What package has an issue

@mantine/tiptap

### Describe the bug

Do we need to install all these ourselves? when following the installation guide after starting the server I kept getting package not found for all of these

```
prosemirror-commands
prosemirror-keymap
prosemirror-model
prosemirror-schema-list
prosemirror-state
prosemirror-transform
prosemirror-view
prosemirror-history
prosemirror-gapcursor
prosemirror-dropcursor
```

### What version of @mantine/hooks page do you have in package.json?

latest

### Do you know how to fix the issue

No

### Are you willing to participate in fixing this issue and create a pull request with the fix

No

### Possible fix

Not sure how it works, don't use peerDeps and install them as regular deps?

**Comment by rtivital** [comment]
It depends on your package manager. You can install these packages if you want. For further installation instructions visit tiptap website.
